### ISSUE TITLE
Can't export all messages containing media to  JSON

### ISSUE THREAD

**Issue by DechanJ** [iyear/tdl#708]
### Describe the bug

When I use "tdl chat export -c CHAT", it only shows 294 media but the channel has 2000+ medias
<img width="281" alt="Snipaste_2024-08-21_21-16-37" src="https://github.com/user-attachments/assets/9994c863-8eda-4cef-95cf-949d1251dc82">
<img width="818" alt="Snipaste_2024-08-21_21-17-46" src="https://github.com/user-attachments/assets/93657f7d-e156-4012-afe9-791c221e93c9">

### To Reproduce

tdl chat export -c CHAT

### Expected behavior

export all messages containing media

### Version

v0.17.3

### Which OS are you running tdl on?

Windows

**Comment by xd003** [comment]
I can also confirm this issue. I just exported last 5 media messages using `tdl chat export -c CHAT -T last -i 5` and it seems to have exported random files from the chat and it's not really the last 5 media so it explains that exporting the whole chat would skip quite a lot of messages

**Comment by iyear** [comment]
I cannot reproduce the issue locally. Please confirm that the channel in the screenshot and the one specified with `-c CHAT` are the same channel.

**Comment by coxfuture** [comment]
@iyear have you tested it with media messages that were forwarded to a group? in my testing, it does not export media from forwarded messages

**Comment by iyear** [comment]
Yes, I've tested it. The forwarded messages are not much different from regular messages, except that there are some values in fields related to forwarding.
